components/features/task-creation/task-create-modal.tsx

```
@@ -0,0 +1,455 @@
+	const [taskData, setTaskData] = useState<TaskData>({
+		title: initialData?.title || "",
+		description: initialData?.description || "",
+		priority: initialData?.priority || "medium",
+		assignee: initialData?.assignee || "",
+		labels: initialData?.labels || [],
+		dueDate: initialData?.dueDate,
+		metadata: initialData?.metadata || { creationMethod: "manual" },
+	});
+
+	// Voice recording state
+	const [isRecording, setIsRecording] = useState(false);
```

Biome (2.1.2)

[error] This variable isRecording is unused.

Unused variables are often the result of an incomplete refactoring, typos, or other sources of bugs.
Unsafe fix: If this is intentional, prepend isRecording with an underscore.


(lint/correctness/noUnusedVariables)